### ISSUE TITLE
(git.install) should install the scalar component by default 

### ISSUE THREAD

**Issue by jackmtpt** [chocolatey-community/chocolatey-packages#2129]
### Checklist

- [X] I have checked the moderation queue that no new version has been submitted (only visible when logged in), or the submitted package version has failed the automated checks.

Please navigate to the following [link](https://community.chocolatey.org/packages?q=&moderatorQueue=true&moderationStatus=all-statuses&prerelease=false&sortOrder=package-download-count) to view the moderation queue.

- [X] I have verified that the new version is a stable release.
- [X] I have looked to see if there are any Bug reports that prevents a new version being submitted and that no existing Outdated Reports have been created.
- [X] I have verified the new version has a Windows binary (EXE or MSI installer, Zip or other archive, etc).

### New Software Version

2.39.1 - this is the currently packaged release.

https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/git.install/tools/helpers.ps1 sets the /COMPONENTS argument automatically, and has no way to specify additional components from what I can see. The `scalar` component should be installed by default so that this new feature of git is available for everyone to use.

Installer argument definition: https://github.com/git-for-windows/build-extra/blob/main/installer/install.iss#L117

**Comment by pauby** [comment]
@jackmtrayport Is this something you can add a PR for?

**Comment by jackmtrayport** [comment]
Unfortunately not

**Comment by pauby** [comment]
> Unfortunately not

Just be aware that this will therefore likely receive little attention.

**Comment by jackmtrayport** [comment]
yes it is still relevant

**Comment by pauby** [comment]
@jackmtrayport We don't let items such as this sit indefinitely. If nobody is willing to pick it up then this shows that there is no interest in adding it, and it will be closed.

**Comment by DarwinJS** [comment]
@pauby - I am interested, but having the time to dig through is something else.

It's ironic that Microsoft contributed this utility that creates clones 90% smaller, but the only OS-Packager combination that does not install it is Chocolatey for Windows :)

I wrote a Git clone super optimization script that is almost working under 'git-bash', except scalar is missing.

https://gitlab.com/gitlab-accelerates-embedded/misc/git-much-faster

I will have to instruct Windows users to use the standard installer rather than chocoloatey.

Would it be possible to include scalar since all the other non-Microsoft OSes include it?

**Comment by pauby** [comment]
@DarwinJS @gep13 this issue has been closed for over 2 years. I've reopened it. I will close [this discussion](https://github.com/orgs/chocolatey-community/discussions/2734#discussioncomment-14551513) in favour of it.

**Comment by gep13** [comment]
@rimrul and just to confirm, the component is simply named "scalar", or is it something else?

Do you happen to know if we are missing other "default" components, due to the nature of how we are installing git?

**Comment by rimrul** [comment]
> [@rimrul](https://github.com/rimrul) and just to confirm, the component is simply named "scalar", or is it something else?

Yes, it's named ["scalar"](https://github.com/git-for-windows/build-extra/blob/825f9ce55af21a433b3619bf6e133084302cbe5c/installer/install.iss#L118).

> Do you happen to know if we are missing other "default" components, due to the nature of how we are installing git?

All other default components are handled. There are a few non-default components that aren't mentioned in `helpers.ps1`.

* `icons\desktop`: this could theoretically get a parameter if someone wants it.
* `consolefont`: this only plays a role on Windows 8.1.
* `autoupdate`: it's probably undesirable for Git for Windows to sidestep chocolatey for updates, so this seems intentional.

**Comment by rimrul** [comment]
> Here is an exact screenshot - includes the defaults. All you have to do is run the installer manually to see them as well.
> 
> https://gitlab.com/gitlab-accelerates-embedded/misc/git-much-faster#windows

That screenshot doesn't accurately reflect the defaults of a fresh installation. `scalar` is enabled by default on a fresh installation and `ext\desktop` is not enabled by default on a fresh installation.

**Comment by gep13** [comment]
@rimrul @DarwinJS thank you both for all the information! This is really helpful!

Are either of you in a position to put in a PR to address the problem?

Having the Chocolatey package installation match the defaults that happen when directly running the application installer makes sense to me.
